src/server/templates/_components/form-multiple-choice.njk

```diff
@@ -53,7 +53,7 @@
 
       <label
         for="{{ inputId }}"
-        class="multiple-choice multiple-choice--{{ variant | default('radio') }}{{ ' multiple-choice--inline' if inline }}{{ ' has-error' if error }}"
+        class="multiple-choice multiple-choise-vertical multiple-choice--{{ variant | default('radio') }}{{ ' multiple-choice--inline' if inline }}{{ ' has-error' if error }}"
```

Comment:
Passing `inline: false` will be enough to display this macro with a vertical/stacked effect.